source/core/src/main/com/csse3200/game/components/quests/QuestManager.java

```diff
@@ -41,150 +39,257 @@ public class QuestManager extends Component {
-        setupPotionsTask(); // Set up potion collection logic here
-        addQuest(guideQuest3);
-        GameState.quests.quests.add(guideQuest3);
-
-        List<Task> listenQuest = new ArrayList<>(List.of(tasks[6]));
-        QuestBasic guideQuest4 = new QuestBasic("Guide's Advice", "Listen to the guide's advice to progress further.", listenQuest, false, null, null, false, false, 0);
-        addQuest(guideQuest4);
-        GameState.quests.quests.add(guideQuest4);
-
-        List<Task> exploreQuest = new ArrayList<>(List.of(tasks[7]));
-        QuestBasic guideQuest5 = new QuestBasic("Exploration", "Explore the kingdom and gather information about Kanga.", exploreQuest, false, null, null, false, false, 0);
-        addQuest(guideQuest5);
-        GameState.quests.quests.add(guideQuest5);
-
-        List<Task> retrieveQuest = new ArrayList<>(List.of(tasks[8]));
-        QuestBasic guideQuest6 = new QuestBasic("Weapon Retrieval", "Retrieve a weapon by completing the snake minigame.", retrieveQuest, false, null, null, false, false, 0);
-        addQuest(guideQuest6);
-        GameState.quests.quests.add(guideQuest6);
-
-        String[] test2StepCompletionTriggers = {"", "spawnKangaBoss"};
-        String[] test2StepTextProg1 = {"Welcome to Animal Kingdom!", "Here let me help with your quest...", "Press Spacebar!"};
-        String[] test2StepTextProg2 = {"Yippeee!", "You completed your Quest!"};
-        List<Task> twoTaskQuestTasks = new ArrayList<>(List.of(tasks[0], tasks[1]));
-        Map<DialogueKey, String[]> test2TaskQuestDialogues = Map.of(
-                new DialogueKey("Cow", 1), test2StepTextProg1,
-                new DialogueKey("Cow", 2), test2StepTextProg2
-        );
-        QuestBasic twoTaskQuest = new QuestBasic("2 Task Quest", "Move then Attack for a Test Quest", twoTaskQuestTasks, false, test2TaskQuestDialogues, test2StepCompletionTriggers, false, false, 0);
-        GameState.quests.quests.add(twoTaskQuest);
 
-        // Create 2 task quest
-        List<Task> finalQuestTasks = new ArrayList<>(List.of(tasks[2], tasks[0], tasks[1]));
-        QuestBasic finalQuest = new QuestBasic("Final Boss", "Complete quest 1 and 2 to summon the boss", finalQuestTasks, false, null, null, false, false, 0);
-        GameState.quests.quests.add(finalQuest);
+    //needs to be changed depending on biome
+    private List<DialogueKey> createQuestDialogues() {
+
+        List<DialogueKey> dialogues = new ArrayList<>();
+
+
+        String[][] cowInitialDialogue = {
+                {"Moo there adventurer, welcome to the Animal Kingdom! I’m your guide. To prove your worth, you’ll need to complete a few tasks. Are you ready?"}
+        };
 
-    }
 
-    /** Creates all tests for quests and dialogues */
-    private void testQuests() {
-        Task[] tasks = createTasks();
-        Map<DialogueKey, String[]> questDialogues = createQuestDialogues();
-        addQuests(tasks, questDialogues);
+        String[][] potionDialogue = {
```

Comment:
A lot of these are never used but I assume it is for future implementation?

Reply:
Removing now :)